archlinux/PKGBUILD-pulseaudio.install

```diff
@@ -0,0 +1,21 @@
+## arg 1:  the new package version
+post_install() {
+  sed -i '/^autospawn/d' /etc/pulse/client.conf
+  echo autospawn=no >> /etc/pulse/client.conf
+
+  ldconfig
+}
+
+post_upgrade() {
+  post_install
+}
+
+## arg 1:  the old package version
+pre_remove() {
+  systemctl disable qubes-gui-agent.service
```

Comment:
This belong to main package, not pulseaudio.

Reply:
Thanks I missed that.